deploy-agent/deployd/common/utils.py

```diff
@@ -219,18 +219,34 @@ def get_info_from_facter(keys):
         log.error("Failed to get info from facter by keys {}".format(keys))
         return None
 
-
-def get_container_health_info(key):
+    
+def get_container_health_info(commit):
     try:
-        log.info(f"Get health info for container {key}")
-        cmd = ['docker', 'inspect', '-f', '{{.State.Health.Status}}', key]
+        log.info(f"Get health info for container with commit {commit}")
+        result = ""
+        cmd = ['docker', 'ps', '--format', '{{.Image}};{{.Names}}']
```

Comment:
can youi include in the PR description the example output of docker ps cmd and docker inspect -f example?

Reply:
I once added some logs in the method. Here is the example:
```
2023-10-18 23:27:41,145 deployd.common.utils:241 INFO Get health info for container with commit c0e2ee123f1fc442a84f94c92263d05c4a29d193
2023-10-18 23:27:41,167 deployd.common.utils:249 INFO for commit c0e2ee123f1fc442a84f94c92263d05c4a29d193: 998131032990.dkr.ecr.us-east-1.amazonaws.com/beacon:c0e2ee123f1fc442a84f94c92263d05c4a29d193;beacon
2023-10-18 23:27:41,183 deployd.common.utils:241 INFO Get health info for container with commit d74918918ff41e3877e0e91a996f049f87b7f75c
2023-10-18 23:27:41,203 deployd.common.utils:241 INFO Get health info for container with commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32
2023-10-18 23:27:41,225 deployd.common.utils:249 INFO for commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32: 998131032990.dkr.ecr.us-east-1.amazonaws.com/sox-traffic-envoy:86d3a57a51f62aa5a03cb66e6c64812e5e386e32;envoy-A
2023-10-18 23:27:41,242 deployd.common.utils:255 INFO for commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32: name is envoy-A
2023-10-18 23:27:41,242 deployd.common.utils:257 INFO for commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32: name is envoy-A, status is b'healthy\n'
2023-10-18 23:27:41,242 deployd.common.utils:249 INFO for commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32: 998131032990.dkr.ecr.us-east-1.amazonaws.com/sox-traffic-envoy-endpoint-register:86d3a57a51f62aa5a03cb66e6c64812e5e386e32;envoy-endpoint-register
2023-10-18 23:27:41,257 deployd.common.utils:255 INFO for commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32: name is envoy-endpoint-register
2023-10-18 23:27:41,257 deployd.common.utils:257 INFO for commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32: name is envoy-endpoint-register, status is b'healthy\n'
2023-10-18 23:27:41,257 deployd.common.utils:249 INFO for commit 86d3a57a51f62aa5a03cb66e6c64812e5e386e32: 998131032990.dkr.ecr.us-east-1.amazonaws.com/sox-traffic-envoy-sds:86d3a57a51f62aa5a03cb66e6c64812e5e386e32;envoy-sds
2023-10-18 23:27:41,271 deployd.common.utils:241 INFO Get health info for container with commit 636c069f9b8f7cada99e87eac77b2b8a04872a18
2023-10-18 23:27:41,289 deployd.common.utils:249 INFO for commit 636c069f9b8f7cada99e87eac77b2b8a04872a18: 998131032990.dkr.ecr.us-east-1.amazonaws.com/helloworlddummyservice:636c069f9b8f7cada99e87eac77b2b8a04872a18;helloworlddummyservice
2023-10-18 23:27:41,303 deployd.common.utils:241 INFO Get health info for container with commit e508d8c9bad67ee4403f818c912a6c24002136f1
2023-10-18 23:27:41,321 deployd.common.utils:249 INFO for commit e508d8c9bad67ee4403f818c912a6c24002136f1: 998131032990.dkr.ecr.us-east-1.amazonaws.com/mcrouter:e508d8c9bad67ee4403f818c912a6c24002136f1;mcrouter
2023-10-18 23:27:41,336 deployd.common.utils:255 INFO for commit e508d8c9bad67ee4403f818c912a6c24002136f1: name is mcrouter
2023-10-18 23:27:41,336 deployd.common.utils:257 INFO for commit e508d8c9bad67ee4403f818c912a6c24002136f1: name is mcrouter, status is b'healthy\n'
2023-10-18 23:27:41,336 deployd.common.utils:241 INFO Get health info for container with commit 8706351475e18caff1f1fdbd8e893e3ccb013bfa
2023-10-18 23:27:41,354 deployd.common.utils:249 INFO for commit 8706351475e18caff1f1fdbd8e893e3ccb013bfa: 998131032990.dkr.ecr.us-east-1.amazonaws.com/metrics-agent:8706351475e18caff1f1fdbd8e893e3ccb013bfa;metrics-agent
2023-10-18 23:27:41,368 deployd.common.utils:241 INFO Get health info for container with commit 34d94ff3700c1144052c69f14e4741a18e3805ab
2023-10-18 23:27:41,386 deployd.common.utils:249 INFO for commit 34d94ff3700c1144052c69f14e4741a18e3805ab: 998131032990.dkr.ecr.us-east-1.amazonaws.com/pastis_proxy:34d94ff3700c1144052c69f14e4741a18e3805ab;pastis_proxy
2023-10-18 23:27:41,401 deployd.common.utils:241 INFO Get health info for container with commit 946cba44b7b6c895a43e314a6ad8748c1221ddd1
2023-10-18 23:27:41,419 deployd.common.utils:249 INFO for commit 946cba44b7b6c895a43e314a6ad8748c1221ddd1: 998131032990.dkr.ecr.us-east-1.amazonaws.com/sox-traffic-ratelimit:946cba44b7b6c895a43e314a6ad8748c1221ddd1;ratelimit-A
2023-10-18 23:27:41,433 deployd.common.utils:255 INFO for commit 946cba44b7b6c895a43e314a6ad8748c1221ddd1: name is ratelimit-A
2023-10-18 23:27:41,433 deployd.common.utils:257 INFO for commit 946cba44b7b6c895a43e314a6ad8748c1221ddd1: name is ratelimit-A, status is b'healthy\n'
2023-10-18 23:27:41,433 deployd.common.utils:241 INFO Get health info for container with commit d5e571fc9694e746e133cc9ef860f0cb4ce9e474
2023-10-18 23:27:41,450 deployd.common.utils:249 INFO for commit d5e571fc9694e746e133cc9ef860f0cb4ce9e474: 998131032990.dkr.ecr.us-east-1.amazonaws.com/singer:d5e571fc9694e746e133cc9ef860f0cb4ce9e474;singer
2023-10-18 23:27:41,465 deployd.common.utils:241 INFO Get health info for container with commit abe3077cb493409e5265de35888389578d39ae9e
2023-10-18 23:27:41,484 deployd.common.utils:249 INFO for commit abe3077cb493409e5265de35888389578d39ae9e: 998131032990.dkr.ecr.us-east-1.amazonaws.com/tcollector:abe3077cb493409e5265de35888389578d39ae9e;tcollector
2023-10-18 23:27:41,500 deployd.common.utils:255 INFO for commit abe3077cb493409e5265de35888389578d39ae9e: name is tcollector
2023-10-18 23:27:41,501 deployd.common.utils:257 INFO for commit abe3077cb493409e5265de35888389578d39ae9e: name is tcollector, status is b'healthy\n'
2023-10-18 23:27:41,501 deployd.common.utils:241 INFO Get health info for container with commit d0eb5f6963ba397b96e39d59d0ab32ab29395d3f
2023-10-18 23:27:41,522 deployd.common.utils:241 INFO Get health info for container with commit 17e4ede82b4e69028a43478a33458efc45e59b03
2023-10-18 23:27:41,542 deployd.common.utils:249 INFO for commit 17e4ede82b4e69028a43478a33458efc45e59b03: 998131032990.dkr.ecr.us-east-1.amazonaws.com/zk-update-monitor:17e4ede82b4e69028a43478a33458efc45e59b03;zk_update_monitor
```